dpctl/tests/test_sycl_device.py

```diff
@@ -0,0 +1,72 @@
+##===------------- test_sycl_device.py - dpctl  -------*- Python -*---------===##
+##
+##                      Data Parallel Control (dpctl)
+##
+## Copyright 2020 Intel Corporation
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##    http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+##
+##===----------------------------------------------------------------------===##
+##
+## \file
+## Defines unit test cases for the SyclDevice classes defined in sycl_core.pyx.
+##===----------------------------------------------------------------------===##
+
+import dpctl
+import unittest
+
+@unittest.skipIf(not dpctl.has_sycl_platforms(), "No SYCL platforms available")
+class TestSyclDevice (unittest.TestCase):
+
+    def test_get_max_compute_units (self):
+        q = dpctl.get_current_queue()
+        try:
```

Comment:
When do we expect an exception? May be just test if  valid value for max_compute_units is returned?